Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/Shadow/DirectionalLightShadow.azsli

```diff
@@ -45,8 +45,12 @@ class DirectionalLightShadow
 
 bool DirectionalLightShadow::UseFullscreenShadows()
 {
+#ifdef FORCE_OPAQUE
+    return true;
+#else
     bool useFullscreenShadows = o_opacity_mode == OpacityMode::Opaque || o_opacity_mode == OpacityMode::Cutout;
     return useFullscreenShadows;
+#endif
```

Comment:
I think after the pipeline abstraction is done we'll be able to just remove the o_opacity_mode shader option.